features/admin.ai.v1/components/branding-card.tsx

```diff
@@ -0,0 +1,185 @@
+    };
+
+    const handleGenerateClick = async () => {
+        await generateAIBrandingPreference(websiteUrl, "carbon.super");
+        setBannerState(BannerState.Collapsed);
+    };
+
+    return (
+        <>
+            <div
+                className={
+                    classNames(
+                        "branding-card",
+                        {
+                            hidden: isGeneratingBranding
+                        },
+                        "branding-card-banner-full"
+                    )
+                }>
+
+                { bannerState === BannerState.Full && (
+                    <Segment
+                        basic
+                    >
+                        <div className="branding-card-banner-full-heading">
+                            <div>
+                                <Header as="h3">{ t("branding:ai.banner.full.heading") }</Header>
+                                <p>{ t("branding:ai.banner.full.subHeading") }</p>
+                            </div>
+                            <Button onClick={ handleExpandClick } color="secondary" variant="outlined">
+                                <GenericIcon
+                                    icon={ AIIcon }
+                                    className="branding-card-banner-full-button"
+                                />
+                                { t("branding:ai.banner.full.button") }
+                            </Button>
+                        </div>
+                    </Segment>
+                ) }
+                { bannerState === BannerState.Input && (
+                    <Segment>
+                        <div className="branding-card-banner-input">
+                            <Icon
+                                name="dropdown"
+                                onClick={ handleCollapseClick }
+                                className="branding-card-banner-input-icon"
+                            />
+
+                            <div className="branding-card-banner-input-content">
+                                <div>
+                                    <Header as="h3" className="branding-card-banner-input-heading">
+                                        { t("branding:ai.banner.input.heading") }
+                                    </Header>
+                                    <p>{ t("branding:ai.banner.input.subHeading") }
+                                        <DocumentationLink
+                                            link={ "develop.applications.editApplication.asgardeoTryitApplication" +
+                                    ".general.learnMore" }
+                                            isLinkRef = { true }
+                                        >
+                                            <Trans
+                                                i18nKey={ "extensions:common.learnMore" }
+                                            >
+                                            Learn More
```

Comment:
add i18n. This should be already available in i18n

Reply:
addressed with https://github.com/wso2/identity-apps/pull/5961/commits/74702057317e243747c3619c370b716c9daea2cf